### PR TITLE
Some nerfs

### PR DESCRIPTION
**Pull request by Aviu00**
## About the PR

Replaced bso's atreides with ~enforcer~ mk58, bso doesn't deserve atreides.
Nerfed n1984 (and mateba) firerate slightly.
Emp implant now only drains 360 energy, but emp now stuns borgs and IPCs and disables energy weapons temporarily (including eswords).
Fixed some mech and emp related bugs.

## Why / Balance

Infinite EMP implant draining batteries fully is not good.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: Aviu
- add: EMP implanter energy drain heavily reduced, but it still disables energy weapons for 10 seconds.
- add: IPCs and borgs are now stunned when affected by EMP.
- add: Eswords can be EMPd now.
- tweak: Removed atreides from bso, replaced it with mk58.
- tweak: Bso now has security survival box.
- tweak: Energy pen is not as good at delimbing anymore.

### CODE REVIEW

**Comment by BombasterDS2** [comment]
Wait, when did BSO got atreides???

**Comment by Aviu00** [comment]
> Wait, when did BSO get atreides???

#1188

Nobody complained though.

**Comment by JustSome1Nearby** [comment]
Imagine not knowing about this change... Sec is chasing you. You take out oll reliable esword and start atacking them. But you ARE getting stunned... You decide to EMP the sec
Your sword is strangely no longer working

Cadet takes out Enforcer, finaly having a reason to use lethals

**Review by Piras314 — APPROVED** [review]

**Comment by Piras314** [comment]
EMP esword is kind of bad but I don't know.

**Comment by Aviu00** [comment]
Clarified emp description in uplink so that it says that eswords can be empd.

**Comment by Aviu00** [comment]
> EMP esword is kind of bad but I don't know.

In some ss13 builds eswords can be empd. It also makes sense. Also eswords are really strong, especially in combination with emp implanter to counter stun batons.

**Comment by BombasterDS2** [comment]
I like idea with EMPable eswords. Don't like that BSO have medium lethal firearms roundstart. 

Also, fix Rempved in changelog

**Comment by Aviu00** [comment]
> Don't like that BSO have medium lethal firearms roundstart.

Why not? HoS has wt550 roundstart, BSO has enforcer. His eg-4 revolver is weak and is useless against esword or eshield users.

**Comment by BombasterDS2** [comment]
> > Don't like that BSO have medium lethal firearms roundstart.
> 
> Why not? HoS has wt550 roundstart, BSO has enforcer. His eg-4 revolver is weak and is useless against esword or eshield users.

HoS is command staff that have access to armory and all sec stuff, that's why he have medium firearm to protect himself. BSO have only department accesses and can easily destroy himself to prevent access leak. Also BSO accompanies another head most of the time that makes stealth attack on head almost impossible (that fits with his game design. He is bodyguard, not military unit)

**Comment by Aviu00** [comment]
> HoS is command staff that have access to armory and all sec stuff, that's why he have medium firearm to protect himself. BSO have only department accesses and can easily destroy himself to prevent access leak. Also BSO accompanies another head most of the time that makes stealth attack on head almost impossible (that fits with his game design. He is bodyguard, not military unit)

Ok, I'll replace it with mk58.

**Comment by Aviu00** [comment]
Also replaced bso's regular survival box with security one. And added missing smoke ingestion blocker component to combat gas masks.

**Comment by a-person5660** [comment]
tbh i dont think theres an issue with giving BSO a nerfed atraides, nor have i seen anyone complain about it.

**Comment by 0tito** [comment]
Emp isnt infinite, it only has 3 uses

File: Content.Server/Power/EntitySystems/BatterySystem.cs [hunk 1]
```diff
@@ -103,6 +105,11 @@ private void CalculateBatteryPrice(EntityUid uid, BatteryComponent component, re
         private void OnEmpPulse(EntityUid uid, BatteryComponent component, ref EmpPulseEvent args)
         {
             args.Affected = true;
+            if (!HasComp<RechargeableBlockingComponent>(uid)) // Goobstation - rechargeable blocking system handles it
+                args.Disabled = true;
+            if (TryComp<MechEquipmentComponent>(uid, out var equipment) &&
+                equipment.EquipmentOwner != null) // Goobstation - it would break mech weapons otherwise
+                return;
```
Comment (gluesniffler) [inline Content.Server/Power/EntitySystems/BatterySystem.cs:112]: Remove this, I'm fixing that issue.

**Comment by Aviu00** [comment]
Made epen not delimb as easily as esword.

**Comment by Gallagin** [comment]
BSO should keep the lower accuracy atreides or get the enforcer at minimum. Nerfing it for the sake of nerfing it when it hasn't been oppressive and no one is complaining about it isn't a good mindset imo

**Comment by whateverusername0** [comment]
> BSO should keep the lower accuracy atreides or get the enforcer at minimum. Nerfing it for the sake of nerfing it when it hasn't been oppressive and no one is complaining about it isn't a good mindset imo

im complaining
BSO does NOT deserve a cc smg

**Review by whateverusername0 — APPROVED** [review]

**Comment by Gallagin** [comment]
> > BSO should keep the lower accuracy atreides or get the enforcer at minimum. Nerfing it for the sake of nerfing it when it hasn't been oppressive and no one is complaining about it isn't a good mindset imo
> 
> im complaining BSO does NOT deserve a cc smg

which is why I'm saying im fine with the enforcer. Just something decent to get around reflects since 90% of the time when people arm up for zombies, blob or whatever BSO is usually later to arm unless they immediately abandon defending cap/bridge, especially with how high the nukie and blob win rates are right now I feel BSO needs another decent weapon on hand in addition to the EG-4 getting reflected in their face. Maybe the atreides was overkill, maybe they'd be better off with a drozd or enforcer, but I don't think a Mk58 is enough

**Comment by lanse12** [comment]
don't give bso shit bruh he has an inf ammo laser gun already

**Comment by whateverusername0** [comment]
> don't give bso shit bruh he has an inf ammo laser gun already

this

**Review by Piras314 — APPROVED** [review]

**Comment by Piras314** [comment]
EMPing esword is gonna be crazy

**Pull request merged**